hal/src/stm32f2xx/pinmap_impl.h

```diff
@@ -56,9 +46,330 @@ extern PinMode HAL_GPIO_Recall_Pin_Mode(uint16_t pin);
 #define ADC_CHANNEL_NONE CHANNEL_NONE
 #define DAC_CHANNEL_NONE CHANNEL_NONE
 
-#ifdef	__cplusplus
-}
-#endif
+#define TIM_PWM_FREQ 500 // 500Hz
+
+#define SERVO_TIM_PWM_FREQ 50 // 50Hz
+
+#if PLATFORM_ID == PLATFORM_PHOTON_PRODUCTION
```

Comment:
Should we move these to platform-specific implementation headers?

Reply:
Fixed, same as ^^